CHANGELOG.rst

```diff
@@ -1,3 +1,5 @@
+.. Copyright (c) 2016-2021 The Regents of the University of Michigan
+.. This file is part of GSD, released under the BSD 2-Clause License.
```

Comment:
I think these files need a newline after the `..` comments.
```suggestion
.. This file is part of GSD, released under the BSD 2-Clause License.

```

Reply:
Yes, I should teach the script to write a blank line after the header if it is not yet present.